src/basic/sigbus.c

```diff
@@ -29,23 +29,29 @@ static void sigbus_push(void *addr) {
 
         /* Find a free place, increase the number of entries and leave, if we can */
         for (size_t u = 0; u < SIGBUS_QUEUE_MAX; u++)
-                if (__sync_bool_compare_and_swap(&sigbus_queue[u], NULL, addr)) {
-                        __sync_fetch_and_add(&n_sigbus_queue, 1);
+                /* OK to initialize this here since we haven't started the atomic ops yet */
+                void *tmp = NULL;
```

Comment:
The `{}` fix from next patch needs to be in this one, really.

Reply:
Oops!  Not sure why my compiler warned for the other one but not for this one.